### ISSUE TITLE
Exchange photos

### ISSUE THREAD

**Issue by DerGut** [codeforosnabrueck/codeforosnabrueck.github.io#2]
Does anybody have good skills with the camera (@a-hain)? We should exchange the photos that are currently on our page for some taken at one of our meetings. But this isn't very urgent.

**Comment by a-hain** [comment]
I can try if you tell me what kind of photos it's supposed to be (like the ones on the page?). The library we're meeting at next time might look a little boring and less inspirational though, especially without daylight. :P But I guess we can discuss this at the meeting

**Comment by DerGut** [comment]
Yeah sure, let's discuss it there. Something like the ones on the page would be cool. But let's see where our next meet ups will take place...
